sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/DoNotHardcodeCredentialsTest.cs

```diff
@@ -43,7 +43,7 @@ public void DoNotHardcodeCredentials_CS_DefaultValues()
         public void DoNotHardcodeCredentials_CS_CustomValues()
         {
             Verifier.VerifyAnalyzer(@"TestCases\DoNotHardcodeCredentials_CustomValues.cs",
-                new CSharp.DoNotHardcodeCredentials(AnalyzerConfiguration.AlwaysEnabled) { CredentialWords = "kode,facal-faire" });
+                new CSharp.DoNotHardcodeCredentials(AnalyzerConfiguration.AlwaysEnabled) { CredentialWords = @"kode,facal-faire,\*+?|{}[]()^$.# " });
```

Comment:
There should be a one test with `\*+?|{}[]()^$.#` value in `DoNotHardcodeCredentials_CustomValues.cs`. In best case, demonstrating {{error message}}.